From: Jiri Benc <jbenc@suse.cz>
Subject: rt2x00: remove duplicate USB ID
Patch-mainline: no
References: 350956

This ID is present in both rt73usb and rt2500usb driver. It is reported by
several people to work with rt73usb and not to work with rt2500usb. Remove
it from the latter one.

Signed-off-by: Jiri Benc <jbenc@suse.cz>

---
 drivers/net/wireless/rt2x00/rt2500usb.c |    1 -
 1 file changed, 1 deletion(-)

--- linux-2.6.23.orig/drivers/net/wireless/rt2x00/rt2500usb.c
+++ linux-2.6.23/drivers/net/wireless/rt2x00/rt2500usb.c
@@ -1786,7 +1786,6 @@ static struct usb_device_id rt2500usb_de
 	/* Ralink */
 	{ USB_DEVICE(0x148f, 0x1706), USB_DEVICE_DATA(&rt2500usb_ops) },
 	{ USB_DEVICE(0x148f, 0x2570), USB_DEVICE_DATA(&rt2500usb_ops) },
-	{ USB_DEVICE(0x148f, 0x2573), USB_DEVICE_DATA(&rt2500usb_ops) },
 	{ USB_DEVICE(0x148f, 0x9020), USB_DEVICE_DATA(&rt2500usb_ops) },
 	/* Siemens */
 	{ USB_DEVICE(0x0681, 0x3c06), USB_DEVICE_DATA(&rt2500usb_ops) },
